README.md

```diff
@@ -25,10 +25,11 @@ and on Git hosting providers like [GitHub](https://github.com/),
 We perform the merge every time you click "merge pull request".
 
 `libgit2` is licensed under a **very permissive license** (GPLv2 with a special
-Linking Exception).  This basically means that you can link it (unmodified)
-with any kind of software without having to release its source code.
-Additionally, the example code has been released to the public domain (see the
-[separate license](examples/COPYING) for more information).
+Linking Exception). The GPLv2 requires you to make changes you make _to libgit2
+itself_ available publicly, but the linking exception means that you can link
```

Comment:
The GPL does not require publishing modifications publicly. it requires you to provide source code to those to whom you have provided the compiled version.

Reply:
True.